.github/workflows/build_wheels.yml

```diff
@@ -61,6 +66,7 @@ jobs:
           CIBW_PRERELEASE_PYTHONS: True
           CIBW_TEST_EXTRAS: test
           CIBW_TEST_COMMAND: pytest {package}/tests
+          CIBW_BUILD_VERBOSITY: 1
```

Comment:
I'm leaving this on so we can quickly check that there are no linkage warnings in the amd64 wheels